Makefile

```diff
@@ -150,7 +150,7 @@ coverage-build: all
 		"$(CURDIR)/testing/coverage/gcovr-patches.diff"); fi
 	if [ -d lib_ ]; then $(RM) -r lib; mv lib_ lib; fi
 	mv lib lib_
-	$(NODE) ./node_modules/.bin/nyc instrument lib_/ lib/
+	$(NODE) ./node_modules/.bin/nyc instrument --extension .js --extension .mjs lib_/ lib/
```

Comment:
will we need similar changes in vcbuild.bat?

Reply:
We don’t have any Windows coverage tooling right now, so: no, but maybe the linter bits?

Reply:
Since even the linter bit is preemptive, IMHO this should not be a blocker.
I'll open a new PR to update the linter bits in `vcbuild`.